perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -38,6 +38,156 @@
 
 namespace map_based_prediction
 {
+
+/**
+ * @brief First order Low pass filtering
+ *
+ * @param prev_y previous filtered value
+ * @param prev_x previous input value
+ * @param x current input value
+ * @param cutoff_freq  cutoff frequency in Hz not rad/s (1/s)
+ * @param sampling_time  sampling time of discrete system (s)
+ *
+ * @return double current filtered value
+ */
+double FirstOrderLowpassFilter(
+  const double prev_y, const double prev_x, const double x, const double sampling_time = 0.1,
+  const double cutoff_freq = 0.1)
+{
+  // Eq:  yn = a yn-1 + b (xn-1 + xn)
+  const double wt = 2.0 * M_PI * cutoff_freq * sampling_time;
+  const double a = (2.0 - wt) / (2.0 + wt);
+  const double b = wt / (2.0 + wt);
+
+  return a * prev_y + b * (prev_x + x);
+}
```

Comment:
Is it difficult to use the lowpass filter in [this package](https://github.com/autowarefoundation/autoware.universe/tree/main/common/signal_processing)?

Reply:
Oh, sorry, I just missed your [comment](https://github.com/autowarefoundation/autoware.universe/pull/2822#discussion_r1107028467). Never mind.